api/src/pcapi/routes/backoffice_v3/utils.py

```diff
@@ -152,3 +152,7 @@ def get_query_params() -> ImmutableMultiDict[str, str]:
 
 def is_feature_active(feature_name: str) -> bool:
     return feature.FeatureToggle[feature_name].is_active()
+
+
+def get_env_var(env_var_name: str) -> typing.Any:
```

Comment:
Je suggère :
```suggestion
def get_setting(env_var_name: str) -> typing.Any:
```
plutôt `get_setting` ou `get_from_settings`, plus générique car en pratique ça prend dans les settings, qui ne sont pas forcément des variables d'environnement (ça peut renvoyer une valeur par défaut en dur, une valeur d'un secret...)